docs/status.md

```diff
@@ -0,0 +1,11 @@
+## Schema Status
+
+The status of each schema in XDM describes the likelihood of change of a schema.
+In particular for XDM Versions prior to XDM 1.0.0, this can inform users and providers of XDM-compatible APIs of the stability of a given schema.
+
```

Comment:
Having this at the schema level is fine, do we need to also have this at the field level when it deviates from the schema's "status"? For example a new field added to a stable schema but that field would not yet be stable.

Reply:
I agree. #303 covers parts of that.